### DISCUSSION TITLE
567A1EC0-8E91-4CE7-989E-5A1E8FCE5F9F

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#141] [General]
# 567A1EC0-8E91-4CE7-989E-5A1E8FCE5F9F

This add-on adds the Cartwheel Galaxy, an unusual ring galaxy in Sculptor. It is a ring galaxy (and a lenticular galaxy), and from Earth it appears nearly face-on. Its shape is unusual in that it looks an (elliptical) wheel, with its 

https://celestia.mobi/addon?item=567A1EC0-8E91-4CE7-989E-5A1E8FCE5F9F&lang=es

**Comment by Pascualito1008** [comment]
oh yeah that was circle hahahahaha SO Big than hoag's object
